DEVELOPMENT.md

```diff
@@ -202,3 +202,48 @@ Additional information which is really important to how the application runs can
 be found in the manifest:
 
 > `$ bundle exec 3scale_backend manifest`
+
+## Running with RubyMine
+
+Using RubyMine scripts can help you run both Worker and Listener.
+
+For that, you'll need to create the following configurations: 
+
+### Worker
+
+First you'll have to set a name. We recommend using `worker`.
+
+Then you'll have to add the following config:
+
+```
+Ruby script: /Users/lvillenj/Projects/apisonator/bin/3scale_backend_worker
```

Comment:
I'd replace your path with some example path, and a comment that this will need to be replaced (in the "parent" section, because it will apply to all configs).

Reply:
Totally correct, I don't know why I skipped this, thank you 🙏